front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -62,6 +62,8 @@
   .scenario-sidemenu {
     padding-top: 0.5rem;
     padding-bottom: 0.5rem;
+    display: flex;
+    flex-direction: column;
```

Comment:
IIRC this ended up causing some issues with scenarii with lots of trains (20 or so). See https://github.com/OpenRailAssociation/osrd/issues/8012

Reply:
Thank you done